scripts/maintenance.awk

```diff
@@ -0,0 +1,15 @@
+## Copyright (C) 1996-2019 The Squid Software Foundation and contributors
```

Comment:
After the "apply all scripts that are named using a hard-coded pattern" change request is implemented, this do-nothing file will not be needed in _this_ PR.